Testing/ORKTest/ORKTest/MainViewController.m

```diff
@@ -1569,6 +1580,73 @@ - (IBAction)showImageChoices:(id)sender {
     [self beginTaskWithIdentifier:ImageChoicesTaskIdentifier];
 }
 
+# pragma mark - Image Capture
+- (id<ORKTask>)makeImageCaptureTask {
+    NSMutableArray *steps = [NSMutableArray new];
+    
+    {
+        ORKInstructionStep *step = [[ORKInstructionStep alloc] initWithIdentifier:@"begin"];
+        step.title = @"Hands";
+        step.image = [UIImage imageNamed:@"hands_solid"];
```

Comment:
Well, thats what I had done in the version I didn't check in.  I presumed there was a good reason why it wasn't made public.  Would like to hear what @jwe-apple has to say on the topic.